lib/ActivityStreams.js

```diff
@@ -182,6 +182,15 @@ ActivityStreams.prototype = {
           SearchProvider.search.performSearch(browser, searchData);
         });
         break;
+      case am.type("NOTIFY_MANAGE_ENGINES"): // eslint-disable-line
```

Comment:
i honestly don't know what it's complaining about - "disallows lexical declarations" (http://eslint.org/docs/rules/no-case-declarations) but i'm not sure why it doesn't like it